app/views/space_invitation_mailer/space_invitation_email.text.erb

```diff
@@ -1,9 +1,9 @@
 Hi <%= @invitation.name %>,
 
-You've been invited to <%= @invitation.space.name %> by <%= @invitation.invitor_display_name %>.
+<%= @invitation.invitor_display_name %> has invited you to join the <%= @invitation.space.name %> Space.
```

Comment:
Let's ditch the word "Space" from this; since the space name should stand alone.

Reply:
fixed in a3ecf1e1bc40279f3745269e09862ab73f1ccad7

Reply:
And removed "the" so it sounds better.
<img width="734" alt="Screenshot 2023-05-22 at 9 03 05 PM" src="https://github.com/zinc-collective/convene/assets/16140873/28468670-8484-4b42-a900-840dd87fac3c">